src/language-js/printer-estree.js

```diff
@@ -2481,7 +2482,10 @@ function printPathNoParens(path, options, print, args) {
               (n.expressions[i].comments && n.expressions[i].comments.length) ||
               n.expressions[i].type === "MemberExpression" ||
               n.expressions[i].type === "OptionalMemberExpression" ||
-              n.expressions[i].type === "ConditionalExpression"
+              n.expressions[i].type === "ConditionalExpression" ||
+              n.expressions[i].type === "LogicalExpression" ||
+              n.expressions[i].type === "BinaryExpression" ||
+              n.expressions[i].type === "TSAsExpression"
```

Comment:
What about `SequenceExpression`?
[Playground link](https://deploy-preview-7010--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEADAjJrWAEAmAwwnAZjPPJwBYbbacBWJ55nANg885wHY--+OACTAAFAEMANDgBGASgC+qANwAdKCEkgIABxgBLaAGdkocQCdzEAO4AFCwhMpxANwj6AJppAzz4sADWcDAAyjr++lAA5sgw5gCucFoAFjAAtgA2AOrJ+vBG4WBwIY55+i55AJ7I4EYmWpFGcOYwtn5RaeLIAGbiGU1aAFZGAB4AQn6BwSHiaXAAMpFwPX0DIMMjIZFRGXAAivEQ8Cv9SSDh5k3mNTLiMnAZ3jrmkTBZnjDJyAAcAAxazwgTSyfh0NWecCuLmWWgAjod4G1dE4QOIjABaKBwOAeHHecxweH6AltcQdLpIXqnLRNNL6WIJM5Gba7A5HZaU1ZnGB3d4eT7IPBaOLifQZbYAYQgaU6NSg0BhIHiTQAKncnFS1i5EgBJKC42AhMAvPQAQX1IRglV2JyaCgUQA)

```js
// Input
`111111111 222222222 333333333 444444444 555555555 666666666 777777777 ${(a, b)}`;

// Output
`111111111 222222222 333333333 444444444 555555555 666666666 777777777 ${(a,
b)}`;

// Expected
`111111111 222222222 333333333 444444444 555555555 666666666 777777777 ${
  (a, b)
}`;
```

Reply:
@sosukesuzuki fixed :+1:

Reply:
hm, i think we should open a issue about it and maybe solve in other a PR, maybe close https://github.com/prettier/prettier/issues/3368 in favor new open an issue, I think there are other places to improve, but you should do this with caution step by step

Reply:
I agree with @evilebottnawi. I think this PR should be tiny fix for #6965.

Reply:
Just because someone complained specifically about logical and binary expressions doesn't mean it shouldn't be fixed for other expressions. E.g. I remembered about `TSAsExpression`, you about `SequenceExpression`, but what about the rest of them? Why is this `if` needed at all? This PR is a perfect opportunity to get rid of it.

Reply:
But okay, let's investigate this later.